db.js

```diff
@@ -28,15 +26,15 @@ class Database {
     }
 }
 
-var db = new Database();
+/*var db = new Database();
```

Comment:
I'd remove this code completely if it's no longer needed rather than commenting out. You can view the commit history if you need to look back at old code.